### PR TITLE
node keys / node address updated

### PR DESCRIPTION
**Pull request by grantnoble**
Updates to original PR #284, based on feedback by @MadelineMurray.

### CODE REVIEW

File: docs/Concepts/Node-Keys.md [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
 
     The private key is not encrypted.
 
 ## Node public key
 
-The node public key displays in the log after starting Besu. Use the
-[`public-key`](../Reference/CLI/CLI-Subcommands.md#public-key) subcommand to export the public key
-to a file.
+The node public key displays in the log after starting Besu. Also refered to as the node ID, the
+node public key forms part the enode URL for a node.
```
Comment (NicolasMassart) [inline docs/Concepts/Node-Keys.md:27]: Maybe the word OF missing in:
"node public key forms part **OF** the enode URL for a node."

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ This command provides node public key related actions.
 
 !!!caution
 
-    To get the public key or address of an existing node, ensure you use the
+    To get the public key or address of a node, ensure you use the
     [`--data-path`](CLI-Syntax.md#data-path) or
-    [`--node-private-key-file`](CLI-Syntax.md#node-private-key-file) option with the `public-key
+    [`--node-private-key-file`](CLI-Syntax.md#node-private-key-file) option with the `public-key`
```
Comment (NicolasMassart) [inline docs/Reference/CLI/CLI-Subcommands.md:49]: Add a link to the command on the `public-key` word?

File: docs/Concepts/Node-Keys.md (hunk 1 above)
Comment (grantnoble) [reply docs/Concepts/Node-Keys.md:27]: Done.

File: docs/Reference/CLI/CLI-Subcommands.md (hunk 2 above)
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Subcommands.md:49]: I didn't put a link on this because it's already inside the command.

**Pull request merged**